openwisp_radius/api/serializers.py

```diff
@@ -98,9 +104,23 @@ def validate(self, data):
             self._disable_token_auth(data['username'])
         return data
 
+    def create(self, validated_data):
+        if app_settings.API_ACCOUNTING_AUTO_GROUP:
+            username = validated_data.get('username', '')
+            try:
+                user = User.objects.get(username=username)
+            except User.DoesNotExist:
+                logging.info(_(f'No corresponding user found for username: {username}'))
```

Comment:
this shouldn't be really happening. I'm not sure why I chose to log it as info.
Can you please change it to `warning` and update the [related docs](https://openwisp-radius.readthedocs.io/en/latest/user/settings.html#openwisp-radius-api-accounting-auto-group)?